src/main.rs

```diff
@@ -25,12 +26,40 @@ arg_enum! {
     }
 }
 
+/// Identify if file is sass entrypoint.
+fn is_entrypoint(entry: &DirEntry) -> bool {
+    matches!(entry.file_name().to_str(), Some(s) if !s.starts_with('_') && is_xcssfile(s))
+}
+
+/// Check if string ends with sass or scss.
+fn is_xcssfile(s: &str) -> bool {
```

Comment:
I'm not sure I understand why this function is named the way it is

Reply:
Because it checks for both `sass` and `scss` so it means `xcss`, `x` means unknown. Maybe you have something better in mind?